client/test/channel.go

```diff
@@ -150,22 +150,21 @@ func (ch *paymentChannel) assertBals(state *channel.State) {
 		ch.bals[0], ch.bals[1],
 		bals[0], bals[1],
 	)
-	assert := assert.New(ch.r.t)
-	assert.Zerof(bals[0].Cmp(ch.bals[0]), "bal[0]: %v != %v", bals[0], ch.bals[0])
-	assert.Zerof(bals[1].Cmp(ch.bals[1]), "bal[1]: %v != %v", bals[1], ch.bals[1])
+	ch.r.RequireTruef(bals[0].Cmp(ch.bals[0]) == 0, "bal[0]: %v != %v", bals[0], ch.bals[0])
```

Comment:
The function is still called `assertBals` even though we are only using `require` instead of `assert` here.

Reply:
I think it is fine. Asserting can also mean that you don't continue afterwards.

Reply:
Hm okay.